main.py

```diff
@@ -27,14 +28,52 @@
 
 class User(db.Model, UserMixin):
     id = db.Column(db.Integer, primary_key=True)
-    name = db.Column(db.String(50))
-    email = db.Column(db.String(50), unique=True)
-    password = db.Column(db.String(80))
+    name = db.Column(db.String(50), nullable=False)
+    email = db.Column(db.String(50), unique=True, nullable=False)
+    password = db.Column(db.String(80), nullable=False)
+    events = db.relationship('Event', backref='user', lazy=True)
+    cameras = db.relationship('Camera', backref='user', lazy=True)
+
+class Event(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    camera_id = db.Column(db.Integer, db.ForeignKey('camera.id'), nullable=False)
+    type = db.Column(db.Enum(EventType), nullable=False)
+    timestamp = db.Column(db.DateTime, nullable=False)
+
+class Camera(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    name = db.Column(db.String(50), nullable=False)
+    status = db.Column(db.Enum(CameraStatus), nullable=False)
+    mode = db.Column(db.Enum(CameraMode), nullable=False)
```

Comment:
Here are the new table models. Some things to note here are the **types** and **foreign key** constraints in some of the columns. I've also updated the database page on our wiki to reflect this.